src/guides/optimism-bridge-widget/index.md

```diff
@@ -0,0 +1,1877 @@
+
+If you want a deeper dive into the bridge contract interfaces and an overview of L1s, L2s, and bridging, catch our presentation at Devcon!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ZpSsFXv7Fwk?start=4292" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+The completed code lives [here](https://github.com/trufflesuite/unleashed_optimism_bridge).
+
+## Bridging basics in the Optimism Bridge Box
+
+In this tutorial, we'll be creating an NFT Marketplace on Optimism and then adding Bridget, an Optimism bridge widget to bridge ETH between Ethereum Goerli and Optimism Goerli to our DApp by building on top of our Optimism bridge
+
+NOTE: To do the bridging for our widget, we will be interacting with the Optimism bridge contract that has already been deployed. Therefore, we will only be interacting with Ethereum Goerli and Optimism Goerli. If we wanted to run all of this locally, there are a lot of additional steps, such as deploying all the Optimism contracts to your local Ethereum and local Optimism and running a local sequencer.
+
+### Download System Requirements
+
+You'll need to install:
+
+- [Truffle](https://trufflesuite.com/docs/truffle/getting-started/installation/)
+
+### Register for a MetaMask wallet
+
+To interact with your DApp in the browser, you'll need a MetaMask wallet. Sign up for an account [here](https://metamask.io/?utm_source=devcon&utm_medium=event&utm_campaign=2022_Oct_optimism-bridge_tutorial_none).
+
+### Download VS Code
+
+Feel free to use whatever IDE you want, but we highly recommend using VS Code! You can run through most of this tutorial using the Truffle extension to create, build, deploy, and debug your smart contracts, all without using the CLI! You can read more about it [here](https://trufflesuite.com/blog/build-on-web3-with-truffle-vs-code-extension/).
+
+### Create an Infura account and Infura project
+
+To connect your DApp to Ethereum Goerli and Optimism Goerli, you'll need an Infura account. Sign up for an account [here](https://infura.io/register?utm_source=devcon&utm_medium=event&utm_campaign=2022_Oct_optimism-bridge_tutorial_none).
+
+Once you're signed in, create a project! Let's call it `optimism-marketplace`, and select Web3 API from the dropdown.
+
+### Get Ethereum Goerli Eth, Optimism Goerli Eth, and Goerli DAI
+
+In order to deploy your contracts and run our bridging scripts, you'll need some test Eth to cover your gas fees! There are multiple ways to get each one:
+
+- Ethereum Goerli Eth
+  - [Paradigm](https://faucet.paradigm.xyz/) has a great MultiFaucet that deposits funds across 8 different networks all at once.
```

Comment:
The indented bullets aren't showing up as indented when they render. As an interim first we could just put them on the same line, something like:

- Ethereum Goerli Eth - [Paradigm](https://faucet.paradigm.xyz/) has a great MultiFaucet that deposits funds across 8 different networks all at once.

Also note the inconsistent full-stops at the end of some of the bulleted lines.